packages/@aws-cdk/aws-events/lib/target.ts

```diff
@@ -41,6 +41,12 @@ export interface RuleTargetConfig {
    */
   readonly role?: iam.IRole;
 
+  /**
+   * The Amazon ECS task definition and task count to use, if the event target
+   * is an Amazon ECS task.
+   */
+  readonly batchParameters?: CfnRule.BatchParametersProperty;
```

Comment:
We don't expose L1 types at the L2 level. Please make an L2 copy of this type.

Reply:
I see I have the wrong comment, but as far as I can tell this change needs to happen in order for an L2 target to be created. Is this incorrect? All of the other property types look to be L1 as well.

Reply:
You are correct good sir!

That is an unfortunate oversight on our part :(. I guess then, in order to be consistent, it would be better to leave it like this.

Reply:
@rix0rrr ah ok, is this something you'd like to change some day?

I've corrected the docs and removed the readme line so this should be good to go.